be/src/olap/olap_cond.cpp

```diff
@@ -382,28 +386,30 @@ int Cond::del_eval(const std::pair<WrapperField*, WrapperField*>& stat) const {
             } else if (stat.first->is_null() && !stat.second->is_null()) {
                 ret = DEL_PARTIAL_SATISFIED;
             } else {
+                DCHECK(false);
                 //不会出现min不为NULL，max为NULL
-                ret = DEL_NOT_SATISFIED;
+                ret = DEL_SATISFIED;
```

Comment:
I think `DEL_NOT_SATISFIED` is more safe. Or you'd better change DCHECK to CHECK